apps/finance/app/src/App.js

```diff
@@ -106,6 +107,7 @@ class App extends React.Component {
               }}
               smallViewPadding={0}
             >
+              {isSyncing && <Badge>Syncing...</Badge>}
```

Comment:
@bpierre @dizzypaty Have we finalized the designs for how the Finance app should show its loading indicator? I imagine we'd want the `FloatIndicator`, but what about the balances and transactions?

Reply:
@sohkai After discussing with @bpierre about the different possibilities here, we arrived to the conclusion that the most practical option for now is to use the `FloatIndicator` for both the initial loading data state (removing the loading table skeleton) and for syncing data after the initial load. 

It's not a good idea to have a custom/different loading screen for each app and it's best to take this opportunity to redesign this component with the new style, layout and find a pattern that can be reused across all apps.